x/oracle/spec/06_params.md

```diff
@@ -0,0 +1,19 @@
+
+<!--
+order: 7
+-->
+
+# Parameters
+
+The oracle module contains the following parameters:
+
+| Key                         | Type         | Example                |
+|-----------------------------|--------------|------------------------|
+| vote_period                 | string (int) | "5"                    |
+| vote_threshold              | string (dec) | "0.500000000000000000" |
+| reward_band                 | string (dec) | "0.020000000000000000" |
+| reward_distribution_ window | string (int) | "5256000"              |
+| accept_list                 | []DenomList  | [{"base_denom": "uumee", symbol_denom": "UMEE", "exponent": "6"}] |
+| slash_fraction              | string (dec) | "0.001000000000000000" |
+| slash_window                | string (int) | "100800"               |
+| min_valid_per_window        | string (int) | "0.050000000000000000" |
```

Comment:
```suggestion
| Key                         | Type         | Example                |
|-----------------------------|--------------|------------------------|
| vote_period                 | uint64 | 5                    |
| vote_threshold              | sdk.Dec | 0.5 |
| reward_band                 | sdk.Dec | 0.02 |
| reward_distribution_ window | uint64 | 5256000              |
| accept_list                 | []DenomList  | [{"base_denom": "uumee", symbol_denom": "UMEE", "exponent": "6"}] |
| slash_fraction              | sdk.Dec | 0.001 |
| slash_window                | uint64) | 100800               |
| min_valid_per_window        | sdk.Dec | 0.05 |
```
Might be best to leave out the string conversions, since the params have number types when in-module.

(I definitely didn't get the spacing right in my suggestion)

Reply:
Wait, why are you suggesting this change? The `keys` are meant to reflect the keys used in state.

Reply:
E.g. it should be `VotePeriod` for the key.

Reply:
Mainly for the `string(int)` to `uint64` and similar in column 2.

Should we basically follow conventions found in places like
https://github.com/cosmos/cosmos-sdk/blob/v0.44.5/x/bank/spec/05_params.md ?

Reply:
Yes 👍